js/base.js

```diff
@@ -60,7 +60,13 @@ export class API extends mix(BaseAPI).with(DocumentAPI, ShipmentAPI) {
     }
 
     async _handleResponse(response, errorMessage = "Problem in request") {
-        const result = await super._handleResponse(response, errorMessage);
+        let result = await response.clone().json();

```

Comment:
It's probably better to override the whole `_handleResponse` than cloning the request just so you can read the buffer twice.